src/datasets/search.py

```diff
@@ -1,3 +1,4 @@
+import importlib
```

Comment:
FYI, to use `importlib.util` you should import it as `import importlib.util`.
I have seen errors on some setups (don't know what causes them exactly, maybe a python version problem?) when not importing the submodule explicitly.

Reply:
thanks, I changed the import to `import importlib.util`